### PR TITLE
Handle numerical problems gracefully in FVA

### PR DESCRIPTION
**Pull request by cdiener**
Addresses #865.

For some models there may exist an optimum for the reference FVA solution but consecutive optimizations may not find the optimum again due to numerical issues, leading to infeasible solutions in FVA loops. This adds code to handle this gracefully. 

I can't really add a test for that since none of our test models are this unstable.

### CODE REVIEW

File: cobra/flux_analysis/variability.py [hunk 1]
```diff
@@ -42,21 +44,35 @@ def _fva_step(reaction_id):
...
+        {rxn.forward_variable: 1, rxn.reverse_variable: -1}
+    )
     _model.slim_optimize()
     sutil.check_solver_status(_model.solver.status)
     if _loopless:
         value = loopless_fva_iter(_model, rxn)
     else:
         value = _model.solver.objective.value
+    # handle infeasible case
+    if value is None:
```
Comment (Midnighter) [inline cobra/flux_analysis/variability.py]: As far as I can tell the value is not set in this condition, so nothing actually changed?

Comment (cdiener) [reply cobra/flux_analysis/variability.py]: Oh yeah, sorry for that. Will fix.
